src/main/java/org/folio/persist/AbstractRepository.java

```diff
@@ -21,19 +21,21 @@
 import io.vertx.sqlclient.Row;
 import io.vertx.sqlclient.RowSet;
 
-abstract class AbstractRepository<T> {
+public abstract class AbstractRepository<T> {
   protected final PostgresClientFuturized postgresClientFuturized;
   protected final PostgresClient postgresClient;
   protected final String tableName;
   protected final Class<T> recordType;
+  protected final String tenantId;
 
   protected AbstractRepository(PostgresClient postgresClient, String tableName,
-    Class<T> recordType) {
+    Class<T> recordType, String tenantId) {
```

Comment:
Why do need `tenantId`? Should not the `postgresClient` know about tenant already?

Reply:
It does now, however the method that returns tenant name has package only access. I've introduced an utility class that is in the same package as `PostgresClient`. Let me know if it is not fine.

Reply:
This is now removed, because a delete method used from postgresClient.